docker-compose.yml

```diff
@@ -138,13 +141,20 @@ services:
       spnet:
 
   influxdb:
-    image: fogsyio/influxdb:1.7
+    image: influxdb:2.6
     environment:
       - INFLUXDB_DATA_ENGINE=tsm1
       - INFLUXDB_REPORTING_DISABLED=false
       - INFLUXDB_ADMIN_ENABLED=true
+      - DOCKER_INFLUXDB_INIT_USERNAME=admin
+      - DOCKER_INFLUXDB_INIT_PASSWORD=sp-admin
+      - DOCKER_INFLUXDB_INIT_ADMIN_TOKEN=sp-admin
+      - DOCKER_INFLUXDB_INIT_ORG=sp
+      - DOCKER_INFLUXDB_INIT_BUCKET=sp
+      - DOCKER_INFLUXDB_INIT_MODE=setup
     volumes:
       - influxdb:/var/lib/influxdb
```

Comment:
Is the old one still required?
Is it used to ensure backwards compatibility?

Reply:
Yes, this is needed for automatic migration (we can't do a fully automated migration since the INIT_MODE must be set to either setup or upgrade).
We can remove the additional volume after the release